packages/textInput/components/TextInputWithBadges.tsx

```diff
@@ -6,7 +6,7 @@ import {
   inputContainer
 } from "../../shared/styles/formStyles";
 import { Badge } from "../../badge";
-import { flex, textTruncate } from "../../shared/styles/styleUtils";
+import { flex, textTruncate, flexItem } from "../../shared/styles/styleUtils";
```

Comment:
The changes in this file are not related to the text input with buttons. I happened to notice two bugs with this component I wanted to fix:
1. It didn't render icons
2. The text input would wrap to a new line when icons were present and the input had a narrow width